identity/app/controllers/editprofile/tabs/SupporterTabs.scala

```diff
@@ -10,8 +10,12 @@ trait SupporterTabs
     extends EditProfileControllerComponents
     with EditProfileFormHandling {
 
-  /** GET /membership/edit */
-  def displayMembershipForm: Action[AnyContent] = displayForm(MembershipEditProfilePage)
+  /** Redirect /membership/edit to manage.theguardian.com/membership */
+  def redirectToManageMembership: Action[AnyContent] = Action { implicit request =>
+    Redirect(
+      url = "https://manage.theguardian.com/membership",
```

Comment:
as per discussion with @walaura I will investigate making this environment specific and implement in a future PR